lib/gen_rmq/consumer/queue_configuration.ex

```diff
@@ -162,11 +81,4 @@ defmodule GenRMQ.Consumer.QueueConfiguration do
         options
     end
   end
-
-  defp set_max_priority_to_highest_value(mp)
-       when is_integer(mp) and mp > @max_priority do
-    255
-  end
-
-  defp set_max_priority_to_highest_value(mp) when is_integer(mp), do: mp
```

Comment:
These functions should be kept and used when setting the max priority using queue arguments